src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.scss

```diff
@@ -100,3 +85,18 @@
   color: $blue;
   cursor: pointer;
 }
+
+// Leave media queries at the bottom of the file to increase css specifity
```

Comment:
`to increase css specifity` -> `to make sure they override CSS rules defined above`

The specificity of the CSS selectors inside of media queries remains the same; that's the whole point of having them at the bottom, meaning that they will override rules from selectors of the same specificity defined above.